docs/pages/machine-id/access-guides/terraform.mdx

```diff
@@ -173,6 +173,91 @@ Check your Teleport cluster, ensuring the role has been created:
 $ tctl get role/terraform-test
 ```
 
+## Example: Github Actions
```

Comment:
Yeah, that makes sense and was kind of in the back of my mind. I think for me the goal is less to get this specific example in and more to make the Machine-ID/Terraform guide more differentiated. What I can do for now is just revert my changes on that doc so we can get the main tutorial in, since that's the one that's broken.

Reply:
I've removed this section. Only left some context on one-shot that I think is actually helpful for the original content.